waterlog/src/Components/Map.js

```diff
@@ -129,8 +133,9 @@ class MapComponent extends Component {
     return (
       <div className="map-main-div">
         <div className="map-tile-div">
-          <Map center={position} zoom={this.state.zoom} zoomControl={false} style={{height:this.props.height}}>
-            {(() => {
+			
+     <Map center={position} maxBounds={[[-25.784510,28.334360],[-25.782110,28.338325]]} zoom={this.state.zoom} zoomControl={false} style={{height:this.props.height}}>
```

Comment:
Why not create a variable for maxBounds as well?

Reply:
Done , please re-check again